src/sas/sascalc/file_converter/nxcansas_writer.py

```diff
@@ -86,9 +86,11 @@ def _write_h5_vector(entry, vector, names=['x_position', 'y_position'],
                 if units is not None:
                     entry[names[2]].attrs['units'] = units
 
-        valid_data = all([issubclass(d.__class__, (Data1D, Data2D)) for d in dataset])
+        valid_data = all([issubclass(d.__class__, (Data1D, Data2D)) for d in
+                          dataset])
```

Comment:
better to break at the `for`:

    valid_data = all([issubclass(d.__class__, (Data1D, Data2D))
                                for d in dataset])

Curious why it doesn't use isinstance?

    valid_data = all([isinstance(d, (Data1D, Data2d)) for d in dataset])